src/ol/interaction/dragrotateinteraction.js

```diff
@@ -117,7 +116,6 @@ ol.interaction.DragRotate.handleDownEvent_ = function(mapBrowserEvent) {
   if (browserEvent.isMouseActionButton() && this.condition_(mapBrowserEvent)) {
     var map = mapBrowserEvent.map;
     map.getView().setHint(ol.ViewHint.INTERACTING, 1);
-    map.render();
```

Comment:
Here too, this seems like it is about communicating the interacting hint.  So I'd want to confirm that the behavior related to pre-render functions is not changed.